ui-core/src/components/MultipleCombobox.tsx

```diff
@@ -3,22 +3,46 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { useVirtualizer } from '@tanstack/react-virtual'
 import classNames from 'classnames'
 import { useCombobox, useMultipleSelection } from 'downshift'
-import React, { useCallback, useMemo, useRef, useState } from 'react'
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
 import { trimStart } from 'lodash'
 
 type MultipleComboboxProps<ComboboxItem> = {
   id?: string
   initialValue?: ComboboxItem[]
   itemToString: (item: ComboboxItem) => string
   options: ComboboxItem[]
```

Comment:
should options be optional now that `choicesByUrl` exists? and perhaps `choicesByUrl` should be `optionsByUrl` for consistency?